package/yast2-vpn.spec

```diff
@@ -27,6 +27,7 @@ Group:          System/YaST
 
 BuildRequires:  yast2
 BuildRequires:  yast2-devtools >= 4.2.2
+BuildRequires:  update-desktop-files
```

Comment:
NP: I'd keep the alphabetical order, i.e. putting `update-desktop-files` at the end. But let's keep it this way...